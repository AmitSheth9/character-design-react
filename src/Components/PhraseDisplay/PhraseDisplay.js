import React from 'react'

export default function PhraseDisplay({phrases}) {
    return (
        <div>
            <div>
                {phrases.map((phrase => {
                    return <p key={phrases.indexOf(phrase)}>{phrase}</p>
                }))}
            </div>
        </div>
    )
}
