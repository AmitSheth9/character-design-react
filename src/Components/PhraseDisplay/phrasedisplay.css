.phrases {
    color: brown;
    font-weight: 800;
    font-size: 20px;
    margin-left: 100px;
}