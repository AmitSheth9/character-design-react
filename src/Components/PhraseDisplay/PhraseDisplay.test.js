import {render, screen} from '@testing-library/react'
import PhraseDisplay from './PhraseDisplay'

it.only('should render our PhraseDisplay component', () => {
    const {container} = render( <PhraseDisplay phrases={['yay', 'test', 'pow']} />

    );
    expect(container).toMatchSnapshot();
})