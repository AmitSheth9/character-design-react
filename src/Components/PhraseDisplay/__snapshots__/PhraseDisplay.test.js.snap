// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render our PhraseDisplay component 1`] = `
<div>
  <div>
    <div>
      <p>
        yay
      </p>
      <p>
        test
      </p>
      <p>
        pow
      </p>
    </div>
  </div>
</div>
`;
